src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/conditions/ExclusionZoneServiceTest.kt

```diff
@@ -41,7 +46,10 @@ class ExclusionZoneServiceTest {
       licenceRepository,
       additionalConditionRepository,
       additionalConditionUploadDetailRepository,
+      documentService,
     )
+
+    Mockito.`when`(documentService.uploadDocument(any(), any())).thenReturn(UUID.randomUUID())
```

Comment:
meh.. usually static import these.. there's a `whenever` you can use to avoid the key word that needs the back ticks. Think you might be importing the non-kotlin version